### PR TITLE
Add aws-kf-ci-bot into the list

### PR DESCRIPTION
**Pull request by PatrickXYS**
This is required as the 1st step to solve this issue: #348 

Note: let me know if you want me to create separate team-list for [aws-kf-ci-bot](https://github.com/aws-kf-ci-bot).

### CODE REVIEW

**Review by Jeffwan — APPROVED** [review]

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -724,6 +725,7 @@ orgs:
             - k8s-ci-robot
             - kubeflow-bot
             - google-oss-robot
+            - aws-kf-ci-bot
```
Comment (Bobgy) [inline github-orgs/kubeflow/org.yaml:728]: /cc @jlewi 
Why does aws-kf-ci-bot need write permissions to these repos?

I think based on discussion in the design doc, it only needs permission to update github PR statuses.
In that case, I'd suggest setting up a new team, probably named like `third-party-bots` and give the minimal permissions it needs.

Comment (PatrickXYS) [reply github-orgs/kubeflow/org.yaml:728]: So there's only admin / write / read access to be granted, I'll make a separate team-list as 'third-party-bots' and start from write permissions to katib / pytorch-operator repo.

**Review by andreyvelich — COMMENTED** [review]
Thanks @PatrickXYS!
/lgtm

File: github-orgs/kubeflow/org.yaml (hunk 1 above)
Comment (Jeffwan) [reply github-orgs/kubeflow/org.yaml]: Prow doesn't support bot to fine grain control the permissions yet. Let's try to make the POC work first and limit the repo it has access to.

**Comment by Jeffwan** [comment]
Once this PR is approved, does Googler need to sync org manually? https://github.com/kubeflow/internal-acls/blob/master/github-orgs/sync_org.sh?

**Comment by Bobgy** [comment]
/lgtm
/approve

Thanks for the update!
I think it's okay for now.

**Comment by Bobgy** [comment]
It will sync automatically

**Pull request merged**